CONTRIBUTING.md

```diff
@@ -0,0 +1,11 @@
+# Contributing
+## Sending a pull request
```

Comment:
There could be a setting up the environment section.
- how to use virtualenv
- how to install legofy locally ```pip install -e .```
- how to use tox for testing locally multiple python versions